django/contrib/contenttypes/models.py

```diff
@@ -137,6 +137,15 @@ def clear_cache(self):
 
     def _add_to_cache(self, using, ct):
         """Insert a ContentType into the cache."""
+        if not isinstance(ct, ContentType):
```

Comment:
I tried to look in `ct._meta`, yes, but couldn't find anything. This seems to do the trick.